resources/ctl.rb

```diff
@@ -39,15 +39,15 @@
   include SensuCookbook::Helpers::SensuCtl
 end
 
-# load_current_value do
-#   cluster_file = '/root/.config/sensu/sensuctl/cluster'
-#   if ::File.exist?(cluster_file)
-#     backend_url JSON.parse(IO.read(cluster_file))['api-url']
-#   end
-# end
-
 action :install do
-  unless platform?('windows')
+  if platform?('windows')
+    include_recipe 'chocolatey'
+
+    chocolatey_package 'sensu-cli' do
+      action :install
+      version new_resource.version unless new_resource.version == 'latest'
```

Comment:
This is technically a breaking change even though I agree its a much better approach. The reasons I consider this breaking:
- previously with no default set you would install `6.1.0` that came from the cookbook, this leads to a consistent install
- if you set a default the interface has changed and without modifications will default to installing "latest"

While this behavior matches the other platforms more closely and I overall agree its a better solution we need to communicate how this impacts the consumers of the cookbook. These changes can only be released as a major change as to ensure the continued operations for those relying on pessimistic version constraints. Please update the changelog with the relevant information.

Reply:
The changelog has been updated. Mind giving it a look and seeing if it needs more?

Reply:
Yes I took a look and it was very helpful, I have a few more questions/clarifications on.